### DISCUSSION TITLE
How to change the decimal dot to a comma?

### DISCUSSION THREAD

**Discussion by BanCrash** [moneymanagerex/moneymanagerex#4180] [Q&A] [answered]
Hi,

I want to know how if there is a way to change the functioning of the prices from be separated by a dot to be separated by a comma, since sometimes I lose the decimals because I write the comma instead the point, and here in Spain we use a comma to separate the values.

I mean, when I write a price transaction, I write "3,50" for example, but money manager replaces with "3.00".

Regards.

**Comment by vomikan** [comment, marked as answer]
There are two options.
1) use locale settings in the options dialog. It effects to global settings.
![image](https://user-images.githubusercontent.com/6836805/152676410-42ee5321-81cb-4d15-ad34-f8115c1c16dc.png)

2) Change the currency settings
![image](https://user-images.githubusercontent.com/6836805/152676474-8f79287f-0728-4ef6-b72b-cbe8c8181424.png)

**Comment by BanCrash** [comment]
@vomikan thank you!! I saw that option but I didn't think that was the answer.

I've also search "decimal" on the wiki but didn't find it neither.
